modules/local/bigstitcher/module/main.nf

```diff
@@ -0,0 +1,70 @@
+process BIGSTITCHER_MODULE {
+    tag { meta.id }
+    container { task.ext.container ?: 'ghcr.io/janeliascicomp/bigstitcher:2.4.1-spark3.2.1-jdk8' }
+    cpus { spark.driver_cores }
+    memory { spark.driver_memory }
+
+    input:
+    tuple val(meta), path(bigstitcher_container), val(spark)
+    val(bigstitcher_container_val)
```

Comment:
@krokicki this API is like that because if an S3 URI it's passed in as path (supported by nf), then nextflow will "stage" the file by copying everything from S3 to the local filesystem. The only way to pass the s3 uri literally is to use val but then paths are not mounted and we need to deal with mounting the volumes

Reply:
Seems fine to have both as options, but shouldn't they be in the same tuple to avoid mapping issues?